Menus/ECommerceSettingsAdminMenu.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Nwazet.Commerce.Permissions;
+using Orchard.Localization;
+using Orchard.UI.Navigation;
+
+namespace Nwazet.Commerce.Menus {
+    public class ECommerceSettingsAdminMenu : INavigationProvider {
+
+        public string MenuName
+        {
+            get { return "admin"; }
+        }
+
+        public ECommerceSettingsAdminMenu() {
+            T = NullLocalizer.Instance;
+        }
+
+        public Localizer T { get; set; }
+
+        public void GetNavigation(NavigationBuilder builder) {
+            builder
+                .AddImageSet("nwazet-commerce")
+                .Add(item => item
```

Comment:
I will change this so the settings are under the global settings section, as is conventional in Orchard, but I nonetheless feel that this is worth discussing as a possible UX improvement.